api/schema_system.go

```diff
@@ -0,0 +1,77 @@
+package api
+
+// SchemaReviewRuleLevel is the error level for schema review rule.
+type SchemaReviewRuleLevel string
+
+// SchemaReviewRuleType is the type of schema rule.
+type SchemaReviewRuleType string
+
+const (
+	// SchemaRuleLevelError is the error level of SchemaReviewRuleLevel.
+	SchemaRuleLevelError SchemaReviewRuleLevel = "ERROR"
+	// SchemaRuleLevelWarning is the warning level of SchemaReviewRuleLevel.
+	SchemaRuleLevelWarning SchemaReviewRuleLevel = "WARNING"
+	// SchemaRuleLevelDisabled is the disabled level of SchemaReviewRuleLevel.
+	SchemaRuleLevelDisabled SchemaReviewRuleLevel = "DISABLED"
+
+	// SchemaRuleMySQLEngine require InnoDB as the storage engine.
+	SchemaRuleMySQLEngine SchemaReviewRuleType = "engine.mysql.use-innodb"
+
```

Comment:
nit: we don't have extra lines for levels above. We should remove the extra empty lines between rule types.

Reply:
Updated.